src/Grid/Grid.js

```diff
@@ -78,8 +81,9 @@ class Grid extends React.Component {
           />
           { body }
           {
-            showFooter === true && 
-              this.props.children
+            footerRenderer &&
+              aggregate &&
```

Comment:
Do you really need to check if the aggregate is there?

Reply:
If there's nothing in the AggregationPayload then the footerRenderer function is not called, I guess